The-Invisibles/nail-salon-frontend
# (ft-scroll-indicator): the setup of the scroll indicator on the wwelcome page

### What does this PR do?
- Implement scroll indicator so that the user's position on the page is always identified.

### How to test the PR?
- Run npm run dev
- Scroll through the homepage and observe the scroll indicator on the right


### Screenshots if any
![welcome](https://user-images.githubusercontent.com/41264503/70561829-7674ed80-1b8b-11ea-8dd5-82c2ffcf2779.PNG)
